### DISCUSSION TITLE
Allow to backup databases that are not in config

### DISCUSSION THREAD

**Discussion by applyACS** [spatie/laravel-backup#1593] [General] [answered]
If I try to run the command only for database but for one that is not in config (backup.source.databases) I get an error that "There are no files to be backed up."

Here is an example:

the command I try to run
```
php artisan backup:run --only-db --db-name=db3
```
the config file

```
'databases' => [
                'db1',
                'db2'
            ],
```

**Comment by angeljqv** [comment, marked as answer]
Feel free to make a PR for that
